src/lib.rs

```diff
@@ -80,33 +80,116 @@ impl PokerRank {
 }
 */
 
-pub fn evaluate(hand: &[Card; 5]) -> i32 {
-    //
-    // existing algorithm in C, but probably will be done differently
-    // https://github.com/theimpossibleastronaut/aa-pokerhands/blob/master/src/functions.c#L155-L187
-    5
+// Evalute for a poker hand.
+//
+//
+pub fn evaluate(hand: &mut [Card; 5]) -> i32 {
```

Comment:
> It might be better if the function returned the enum rather than an integer? (Or more Rust-style.)

Yeah, it'll ultimately be something else. As we discussed, to break ties, we need to know some specifics about the ranks, etc.